Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchableVolume.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
```

Comment:
@wiwei I wish we had a way to mark this component as "BETA" or "EXPERIMENTAL". It will be the first time we are showing it in a release and I could see the component becoming subject to change as people put it through its paces.